### DISCUSSION TITLE
Idea for species informational page

### DISCUSSION THREAD

**Discussion by rory-spurr** [rory-spurr/ESAPermitsCapstone#28] [Ideas]
What if we have a page describing the ESA-listed species seen in our data? We could talk about conservation concerns and research to help conservation. Might give a bit more context to our data and app. Thoughts?

**Comment by DianaDishman-NOAA** [comment]
Thanks Rory! I think people would be interested to see a list of species in the data, although it is almost always going to match the list of the 28 listed ESUs/DPSs of salmon and steelhead in the WCR and the 2 rockfish/1 sturgeon/1 eulachon DPSs.  So rather than derive any lists from the input data, this page could be a helpful place to refer people back to the listed species info that already exists on NOAA's website, but in a really clear plain language. So making the resources super easy to find. For example, include a static list of the WCR listed species, and then have sections of where to go for more info, and I can provide any helpful links. 
Some draft text to start working with: "Want to know more about the biology of these species? Look here [link to species info pages]" "Want to know how these species are doing? Look here [link to viability assessments]" And then for research, something like "Why is research needed on these listed species?  The amount of information we have on how these species are doing is really variable.  In some cases we know a lot about how many fish there are and where they go [link report].  In others, we are lacking key information, which can be because it's difficult to do research at the right times or places to collect data [example webstory], we don't know what's happening to fish as they migrate between places they are counted [webstory], or even simply that their numbers are so low we rarely see them so we don't have much data to work with [webstory]."
"What research is needed for these species?  Our scientists created lists of research needed to recover listed species at the time we drafted their recovery plans [links].  New research needed to address emerging problems (such as wildfires or chemicals that rain washes off of urban roads and into streams) is called out in the most recent 5-year reviews [links]" 
Make sense? Anne or Alana can weigh in too, but I really don't want you to spend time re-drafting we already have elsewhere that might just be difficult to find on NOAA's website.
